spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -1071,6 +1072,13 @@ public void customize(Context context) {
 			 */
 			private boolean buffered = true;
 
+			/**
+			 * Customized date format in the access log file name. <a href=
+			 * "http://tomcat.apache.org/tomcat-7.0-doc/config/valve.html#Access_Log_Valve">Tomcat
```

Comment:
You cannot use HTML in field javadoc as we use that to generate description for the configuration key. Please look at other keys in that file for more inspiration.